packages/docker/manifest.yml

```
@@ -1,6 +1,6 @@
 name: docker
 title: Docker Metrics
-version: 2.1.0
+version: 2.1.1
```

This includes a new feature apart of a bugfix, so maybe this should be a new minor. This also gives margin in case there is a need to release more bugfixes for 8.0/8.1, as 2.1.X versions.
```suggestion
version: 2.2.0
```